### PR TITLE
Add blog about Das Krankenhaus magazine article

### LINKED ISSUE #1722: Add user case
@lennartdefacto can this be formed into a user case for the website? Please in the same format as the other user cases. If you need more info let me know.

It shouldn't go online before April though!!

[Anwenderbericht_Das Krankenhaus_frei Geringhoff.docx](https://github.com/DefactoSoftware/website/files/2946215/Anwenderbericht_Das.Krankenhaus_frei.Geringhoff.docx)

### PR DESCRIPTION
**Pull request by lennartdefacto**
- [x] create blogpost using input from #1722 
- [x] feature blog on homepage (both news and regular featured)
- [x] add news, featured, mood and social images
- [x] remove old unused news image

close #1722

### CODE REVIEW

**Comment by lennartdefacto** [comment]
@isabelamy Could you state what you'd like as titles and if there are other things you'd want changed (images/links etc)?

**Comment by RonaldDijkstra** [comment]
You should link to the deploy review of the blog ;-)

**Comment by lennartdefacto** [comment]
[Blog page for the Krankenhaus article](https://deploy-preview-1857--defacto-de.netlify.com/blog/mehr-qualitat-in-der-patientenversorgung/)

File: data/news_de.yml [hunk 1]
```diff
@@ -9,9 +15,3 @@
   image: "news/news-audit.jpg"
   link: "/blog/audit-pharmaunternehmen-pohl-boskamp/"
   alt: "Audit von Pharmaunternehmen Pohl Boskamp mit Bravour bestanden"
-
-- title: "Defacto gewinnt die Sana AG als neuen Kunden"
-  text: "Sana Kliniken AG entscheidet sich im Rahmen des Projektes zur Einführung einer übergeordneten E-Learning Lösung für das Lernmanagementsystem, CAPP LMS, von Defacto."
-  image: "news/sana-news.jpg"
```
Comment (RonaldDijkstra) [inline data/news_de.yml]: Remove image please

Comment (lennartdefacto) [reply data/news_de.yml]: Done.

**Comment by RonaldDijkstra** [comment]
Ready?

**Comment by lennartdefacto** [comment]
Ready for final review by @isabelamy:
This is the [Blogpage](https://deploy-preview-1857--defacto-de.netlify.com/).

**Comment by RonaldDijkstra** [comment]
I approve the technical side of this.

**Comment by lennartdefacto** [comment]
- ./build/welk-lms-past-bij-mij/resultaten/index.html
  *  External link https://www.defacto.nl/welk-lms-past-bij-mij/resultaten/ failed: 404 No error

It didn't show this error before, does it need attention? @RonaldDijkstra

**Comment by RonaldDijkstra** [comment]
> It didn't show this error before, does it need attention? @RonaldDijkstra

No, you changed the path of the LMS questionnaire and that's not yet in this branch. So the 404 is okay.

**Pull request merged**